roles/etcd/tasks/gen_certs_script.yml

```diff
@@ -68,7 +68,6 @@
   delegate_to: "{{groups['etcd'][0]}}"
   when:
     - gen_certs|default(false)
-    - inventory_hostname == groups['etcd'][0]
```

Comment:
That will just cause the script to run more times. If there's a bug, it is in setting gen_node_certs for all hosts.

Reply:
Scale playbook run tasks only on kube-node hosts. In this case, condition: inventory_hostname == groups['etcd'][0] will always be false.

Reply:
This runs on etcd hosts too?

Reply:
Example of my inventory:
```
[kube-master]
node1
node2
node3

[kube-node]
node4
node5
...

[etcd]
node1
node2
node3
```

In my case cluster scale tasks not runs on ETCD hosts.

Reply:
The fix works for me, we've been able to scale the cluster. @mattymo why do you say that the script would be played more times with your fix?

For me , the `run_once` paired with the `delegate_to` that ensure it is played on `groups['etcd'][0]` is enough to make sure it won't be played multiple times on both cluster.yml and scale.yml.

Reply:
This fix worked for me as well